app/js/daemonManager.js

```diff
@@ -96,19 +100,13 @@ function DaemonManager() {
 		}, function() {
 			UI.notify('Starting siad...', 'start');
 
-			// daemon as a background process logs output to files
-			var out = Fs.openSync(Path.join(__dirname, siaPath, 'daemonOut.log'), 'w');
-			var err = Fs.openSync(Path.join(__dirname, siaPath, 'daemonErr.log'), 'w');
```

Comment:
is there somewhere else that the files go? I feel like these files would be really useful to have, though I forgot about them and haven't asked users for them yet.